solace-deployment.yml

```diff
@@ -95,6 +95,10 @@ instance_groups:
         username_admin_password: ((vmr_admin_password))
         username_mgmt_globalaccesslevel: admin
         username_mgmt_password: ((mgmt_password))
+        username_monitor_globalaccesslevel: ((username_monitor_globalaccesslevel))
```

Comment:
since this is a static (non user configurable), you could hardcode it and that would allow you to not handle that value in bosh-lite-testing.sh